pkgs/applications/version-management/git-and-tools/stgit/default.nix

```diff
@@ -1,11 +1,11 @@
-{ stdenv, fetchFromGitHub, python2, git }:
+{ stdenv, python3, python3Packages, fetchFromGitHub, git }:
 
 let
   name = "stgit-${version}";
   version = "0.22";
 in
-stdenv.mkDerivation {
-  inherit name;
+python3Packages.buildPythonApplication rec {
+   inherit name;
```

Comment:
can you instead use `pname` and `version` attributes?

Reply:
ok, done